staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas.go

```diff
@@ -106,14 +145,34 @@ func SchemaDeclType(s *schema.Structural, isResourceRoot bool) *DeclType {
 		if s.ValueValidation != nil {
 			switch s.ValueValidation.Format {
 			case "byte":
```

Comment:
this branch needs test coverage (see with `go test k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model -coverprofile=coverage.out && go tool cover -html=coverage.out`)

Reply:
Gotcha, fixed (thanks for letting me know about the coverage tool as well!)